columnflow/columnar_util.py

```diff
@@ -3045,38 +3049,39 @@ def open_coffea_root(
         else:
             raise Exception(f"'{source}' cannot be opened as coffea_root")
 
-        return (source, tree.num_entries)
+        return ((source, tree_name), tree.num_entries)
```

Comment:
It is now required to pass the tree name in the coffea events factory, so the name that is identified here, should be returned and reused by the read method below.